src/meta-srv/src/metasrv/builder.rs

```diff
@@ -328,8 +341,7 @@ fn build_ddl_manager(
     procedure_manager: &ProcedureManagerRef,
     mailbox: &MailboxRef,
     table_metadata_manager: &TableMetadataManagerRef,
-    (selector, selector_ctx): (&SelectorRef, &SelectorContext),
-    table_id_sequence: &SequenceRef,
+    table_meta_allocator: TableMetadataAllocatorRef,
```

Comment:
```suggestion
    table_metadata_allocator: TableMetadataAllocatorRef,
```
Let's make the codebase have consistent styles.